variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactoryTest.java

```diff
@@ -551,32 +552,31 @@ public void testParseInfo() {
     @Test
     public void testVariantIds() {
 
+        Set<String> emptySet = new HashSet<>();
         // test that an ID is properly handled
         String line = "1\t1000\trs123\tC\tT\t.\t.\t.";
         List<Variant> expResult = new LinkedList<>();
+        //EVA-942 - Since we ignore IDs submitted through VCF, they are no longer part of the expected result
```

Comment:
as now there is no code using the ids, this comment might get lost, I would put it by the assertion, which is the line of code that might raise the question "why the ids should be empty?" and not this line.